can/interfaces/slcan.py

```diff
@@ -86,8 +84,8 @@ def __init__(self, channel, ttyBaudrate=115200, timeout=1, bitrate=None, **kwarg
     def write(self, string):
         if not string.endswith('\r'):
             string += '\r'
-        self.serialPort.write(string.decode())
-        self.serialPort.flush()
+        self.serialPortOrig.write(bytes(string,'utf-8'))
```

Comment:
Not sure this will work for Python 2.7.

```
>>> bytes('O', 'utf-8')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: str() takes at most 1 argument (2 given)
```

Although it should work with just `bytes(string)` as all characters used are plain ASCII.